Maps/ArchipelagoCampaign/WoL/ap_devils_playground.SC2Map/MapScript.galaxy

```diff
@@ -5009,6 +5207,111 @@ void gt_ArchInit_Init () {
+        UnitRemove(UnitFromId(1433416105));
+        UnitRemove(UnitFromId(1016016046));
+        lib15EF4C78_gf_RemoveUnusedStartingUnits(UnitGroupFromId(5), "Terr", false);
+    }
+    else {
+        lib15EF4C78_gf_RemoveUnusedStartingUnits(UnitGroupFromId(5), libABFE498B_gv_aP_Triggers_Option_overridePlayerRace, false);
+        autoB9B7EC79_g = UnitGroup("AP_Reaper", gv_p05_TOSH, RegionEntireMap(), UnitFilter(0, 0, (1 << c_targetFilterMissile), (1 << (c_targetFilterDead - 32)) | (1 << (c_targetFilterHidden - 32))), 0);
+        autoB9B7EC79_u = UnitGroupCount(autoB9B7EC79_g, c_unitCountAll);
+        for (;; autoB9B7EC79_u -= 1) {
+            autoB9B7EC79_var = UnitGroupUnitFromEnd(autoB9B7EC79_g, autoB9B7EC79_u);
+            if (autoB9B7EC79_var == null) { break; }
+            UnitRemove(autoB9B7EC79_var);
+        }
+        TechTreeAbilityAllow(1, AbilityCommand("AP_CommandCenterTrain", 0), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 13), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 18), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 3), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 6), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 22), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 0), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 20), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 24), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 27), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 26), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 25), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 4), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 10), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 15), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 21), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 12), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 5), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 19), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 7), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 8), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 9), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 11), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 1), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 23), false);
+        TechTreeAbilityAllow(1, AbilityCommand("AP_TerranBuild", 2), false);

```

Comment:
That won't work as unlocks will restore that (try getting a Terran item like Marine and grab a location)

Reply:
Maybe simply just allow Tosh miners to build like rebels can in Outlaws

Reply:
If you want to disable building, you can add a behavior that disables abil classes (`CAbilBuild` and `CAbilTrain`)

Reply:
Good catch, thanks. Fixed: 
![image](https://github.com/user-attachments/assets/9ad1916f-e465-4f82-a315-0ccbe3773552)

I found that if the SCVs could build, then the mission basically just became the Terran version again. Adding the behavior and changing the models makes it a little clearer that these are "special" units rather than another build tree to access.